pkg/clioptions/clusterdiscovery/csi.go

```diff
@@ -6,23 +6,51 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/openshift/origin/test/extended/storage/csi"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/kubernetes/test/e2e/framework/testfiles"
 	"k8s.io/kubernetes/test/e2e/storage/external"
+	"sigs.k8s.io/yaml"
 )
 
 const (
 	CSIManifestEnvVar = "TEST_CSI_DRIVER_FILES"
+	OCPManifestEnvVar = "TEST_OCP_DRIVER_FILES"
 )
 
 // Initialize openshift/csi suite, i.e. define CSI tests from TEST_CSI_DRIVER_FILES.
 func initCSITests(dryRun bool) error {
-	manifestList := os.Getenv(CSIManifestEnvVar)
-	if manifestList != "" {
-		manifests := strings.Split(manifestList, ",")
+	ocpDrivers := sets.New[string]()
```

Comment:
BTW, I think we're not using `dryRun` anywhere.

Reply:
Removed as a new commit.